### PR TITLE
Update: Some titles improvements

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-170](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-170)

## What?
Add member name in Claim Details title
Add title in Conversations

## Why?
For a greater understanding of what is your location

## Optional screenshots
image
image

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/features/claims/claim-details/components/MemberInformation.tsx [hunk 1]
```diff
@@ -113,6 +114,18 @@ export const MemberInformation: React.FC<{
     },
   ])
 
+  const nameEndsOnS =
+    member?.firstName?.split('')[member?.firstName?.length - 1] === 's'
```
Comment (rasmusguterstam) [inline src/features/claims/claim-details/components/MemberInformation.tsx:118]: What's the point of `split('')`?

File: src/features/claims/claim-details/components/MemberInformation.tsx [hunk 2]
```diff
@@ -113,6 +114,18 @@ export const MemberInformation: React.FC<{
...
+
+  useTitle(
+    member && member.firstName
+      ? !nameEndsOnS
+        ? `${member?.firstName}’s Claim`
+        : `${member?.firstName}' Claim`
```
Comment (rasmusguterstam) [inline src/features/claims/claim-details/components/MemberInformation.tsx:124]:
```suggestion
        ? `${member?.firstName}’s claim`
        : `${member?.firstName}' claim`
```

**Review by rasmusguterstam — APPROVED** [review]
🤘

File: src/features/claims/claim-details/components/MemberInformation.tsx (hunk 1 above)
Comment (maxplt) [reply src/features/claims/claim-details/components/MemberInformation.tsx:118]: Oops. Sry, I did strange stuff 😅

**Pull request merged**